ApexOSA/osafs.py

```
@@ -0,0 +1,333 @@
+from .Common import Send, Receive
+import sys
+from .Errors import *
```

Ruff (0.12.2)

`from .Errors import *` used; unable to detect undefined names

(F403)